riak_testing/tasks/main.yml

```diff
@@ -1,16 +1,16 @@
 ---
-- name: ensure a bash_profile file exists
+- name: ensure a bash_profile file exists for user
   file: "dest={{ lookup('env','HOME') }}/.bash_profile state=touch"
```

Comment:
this task ensures bash_profile is present but I think bash_profle is needed before this step, and so it would be best to create bash_profile in bootstrap.sh I believe.